.travis.yml

```diff
@@ -1,23 +1,37 @@
+sudo: required
+services:
+  - docker
 language: c
-install: wget https://raw.githubusercontent.com/avsm/ocaml-travisci-skeleton/mirage3/.travis-mirage.sh
-script: bash -ex .travis-mirage.sh
+cache: false
+
+before_install:
+  - sudo apt-get -qq update
+  - sudo apt-get install -y realpath # needed on ubuntu 14.04.5 LTS / trusty
+  - wget https://raw.githubusercontent.com/mor1/dommage/master/dommage
```

Comment:
Perhaps we should pin a commit (or release tag) here rather than tracking master?